webservices/resources/committees.py

```diff
@@ -74,6 +74,11 @@ def get_committees(self, kwargs):
         if kwargs['cycle']:
             committees = committees.filter(models.Committee.cycles.overlap(kwargs['cycle']))
 
+        if kwargs['start_date']:
+            committees = committees.filter(models.Committee.first_file_date >= kwargs['start_date'])
+        if kwargs['end_date']:
```

Comment:
We could go for something more verbose, like `max_first_file_date`. What do think?